### ISSUE TITLE
Unable to set configuration parameters of Z-wave devices from flow + set parameters to 0

### ISSUE THREAD

**Issue by TedTolboom** [athombv/homey#506]
**What is your Homey version (Settings → System)?**
0.8.30

**What did you try to achieve?**
In relation to #464 tried to set the Z-wave configuration parameters of a Fibaro Motion Sensor v2.7 from a Flow
<img width="1193" alt="schermafbeelding 2016-04-24 om 09 07 25" src="https://cloud.githubusercontent.com/assets/17163431/14766246/9cdc667c-0a02-11e6-9292-303f5eb30481.png">

**What was the result?**
1) No parameters updated; behavior of the sensor is not changed (even after wakeup interval)
    So even the parameters 80 (LED) and 89 (Tamper) which I could not set due to 3) are not updated
2) parameters are not shown in the device settings under Homey Zones & Devices
<img width="626" alt="schermafbeelding 2016-04-24 om 09 07 59" src="https://cloud.githubusercontent.com/assets/17163431/14766277/71b7ca8a-0a03-11e6-8488-9594880aedb4.png">
3) not able to set a parameter to 0. Flow card updates 0 automatically to 1.

"Set configuration parameters" apparently not working at all

**What did you expect as result?**
1) Parameters to be updated
2) Parameters to become visible after a flow has been executed in the device settings
3) able to set a parameter to 0 (e.g. to disable reporting / LED)

**Any other remarks**

...

**Comment by ocatie** [comment]
1) If you send a log I'll take a look to see why the settings aren't applied. Keep in mind that the values are queued for battery powered devices and are applied when the device wakes up.

2) It is true you wont see the settings applied in your flow listed in your device settings. It's on the todo list but not high priority.

3) Thanks for mentioning. Will fix.

**Comment by TedTolboom** [comment]
@ocatie Just tested it with setting 80 (LED indicator)... it is possible to set it to 10 and 11... and updates almost instantaniously ... as shown in below graph (illumination update)
<img width="1163" alt="schermafbeelding 2016-04-29 om 09 32 55" src="https://cloud.githubusercontent.com/assets/17163431/14910449/9f845fbc-0ded-11e6-9c95-a32cb90ffe33.png">

LOG file: C375FE7935

**Comment by ocatie** [comment]
I don't see the device waking up in the log. Because it is a battery powered device the device has to wake up in order for the settings to be applied.

**Comment by TedTolboom** [comment]
After initiating a setting change by a Flow, the motion sensor does not seem to be updating the parameters. Even with a manual wakeup (3x B-button)..

The LED of the motion sensor keeps flashing upon movement... even though I set it to 0..
<img width="1165" alt="schermafbeelding 2016-05-04 om 21 56 37" src="https://cloud.githubusercontent.com/assets/17163431/15027345/9f1c1eca-1243-11e6-834a-6ab54d25306b.png">

LOG: 58D851F458

**Comment by bvdbos** [comment]
setting to 0 isn't possible yet as Taco stated this will be fixed. setting parameter 80 in a flow works for me.

**Comment by TedTolboom** [comment]
@bvdbos I know.
What I meant is that after trying to update the parameters of the flow sensor via a flow (in this case 40 and 42), I tried to update the parameters (after several hours / the wakeup interval) via the device card.

I tried multiple times, including manual wake ups, but it did not update the parameter settings (40 and 42) and also not the parameters I use to get visual confirmation of the update (80 and 89).. The LED kept blinking upon motion while I set it to 0 (LED off)... Restart of Z-wave chip / Homey reboot did not work / resolve this issue..

Some more ASCII's, hope it explains the issue better now. ;)

Now I removed the motion sensor, reset it to defaults and re-added it and it is possible to update it.
